app/AccountancyModule/AccountancyHelpers.php

```diff
@@ -306,6 +306,15 @@ public static function dateRange(array $dates): string
         }
 
         [$start, $end] = $dates;
+
+        if ($end === null) {
+            if ($start === null) {
+                return 'Žádné datumy nejsou k dispozici';
```

Comment:
data, ne datumy :D

Reply:
Ještě jsem si to právě předtím hledal a datumy jsou přípustné. https://www.pravopisne.cz/2010/11/data-x-datumy/ Jde o to, že já chci aby bylo jasné že nechybí nějaká obecná data o akci `Žádna data nejsou k dispozici` ale konkrétna datum od i do. Obecně je to korner case, který by něměl moc nastávat.

Reply:
I stand corrected.